spec/ravelin/supplier_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe Ravelin::Supplier do
```

Comment:
Since we're overriding the `home_location` accessor, we should have a test for it (it would catch the currently existing error 😉 )